airflow/jobs/scheduler_job.py

```diff
@@ -1140,8 +1140,8 @@ def _emit_pool_metrics(self, session: Session = None) -> None:
         pools = models.Pool.slots_stats(session=session)
         for pool_name, slot_stats in pools.items():
             Stats.gauge(f'pool.open_slots.{pool_name}', slot_stats["open"])
-            Stats.gauge(f'pool.queued_slots.{pool_name}', slot_stats[TaskInstanceState.QUEUED])
-            Stats.gauge(f'pool.running_slots.{pool_name}', slot_stats[TaskInstanceState.RUNNING])
+            Stats.gauge(f'pool.queued_slots.{pool_name}', slot_stats["queued"])
```

Comment:
Maybe `TaskInstanceState.QUEUED.value` is better? (Same for below)

Reply:
Added a separate enum for that .